---
title: "Human mtDNA Analysis: FASTQ Alignment"
author: "Your Name"
date: "`r Sys.Date()`"
output: html_document
---
## 1. Introduction

This document demonstrates alignment of raw human NGS reads to the mitochondrial reference genome and outlines variant calling for mtDNA using modern tools.

## 2. Setup

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
library(Rsubread)
library(QuasR)
library(GenomicAlignments)
library(VariantAnnotation)
library(ggplot2)
library(data.table)
```

## 3. Download/Specify Inputs

FASTQ files and reference genome should be present in the working directory.

```{r download}
fastq_file <- 'SRR062634.filt.fastq.gz'
ref_fasta <- 'GRCh38.fa'
```

## 4. Alignment

```{r alignment, echo=FALSE}
buildindex(basename="GRCh38_index", reference=ref_fasta)
align(index="GRCh38_index", readfile1=fastq_file, output_file="aligned.bam")
```

## 5. BAM Summary and Visualization

```{r plot, echo=FALSE}
summary <- qQC("aligned.bam", genome=ref_fasta)
plot(summary$coverage)
```
